debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/Lsn.java

```diff
@@ -144,7 +162,7 @@ public boolean isValid() {
 
     @Override
     public String toString() {
-        return "LSN{" + asLong() + '}';
+        return "LSN{" + asUnsignedBigInteger().toString() + '}';
```

Comment:
This might not be needed. But then we will need to find another way of representation without the user getting negative values because that doesn't sound like a good user experience.